exp/controllers/azureasomanagedcontrolplane_controller.go

```diff
@@ -220,6 +223,9 @@ func (r *AzureASOManagedControlPlaneReconciler) reconcileNormal(ctx context.Cont
 		return ctrl.Result{}, fmt.Errorf("failed to reconcile kubeconfig: %w", err)
 	}
 
+	asoManagedControlPlane.Status.Ready = !asoManagedControlPlane.Status.ControlPlaneEndpoint.IsZero()
+	asoManagedControlPlane.Status.Initialized = asoManagedControlPlane.Status.Ready
```

Comment:
This transitive assignment suggests that Ready and Initialized are equivalent. The comments above suggest otherwise.

```
// Initialized represents whether or not the API server has been provisioned.

// Ready represents whether or not the API server is ready to receive requests.
```

Reply:
Also clarified here in the controller code.